templates/layouts/layout.html.twig

```diff
@@ -257,7 +312,11 @@
             }
         </script>
 
-        <script type="text/javascript" src="https://translate.google.com/translate_a/element.js?cb=googleTranslateElementInit" async></script>
+        <script
+            type="text/javascript"
+            src="https://translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"
```

Comment:
When I try to add the integrity and crossorigin params on this asset, it fails with a CORS error about AllowedOrigin.

Reply:
It's probably not a good idea to add integrity for non-versioned resource anyway. Ad crossorigin, isn't it because the script needs cookies?